src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -275,6 +275,31 @@ ucs_status_t uct_rc_verbs_ep_am_short(uct_ep_h tl_ep, uint8_t id, uint64_t hdr,
     return UCS_OK;
 }
 
+ucs_status_t uct_rc_verbs_ep_am_short_iov(uct_ep_h tl_ep, uint8_t id,
+                                          const uct_iov_t *iov, size_t iovcnt)
+{
+    uct_rc_verbs_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_rc_verbs_iface_t);
+    uct_rc_verbs_ep_t *ep       = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
+    uint32_t length             = uct_iov_total_length(iov, iovcnt);
+    struct ibv_sge sge[UCT_IB_MAX_IOV];
+    struct ibv_send_wr wr;
```

Comment:
can we re-use `inl_am_wr` + `inl_sge[UCT_IB_MAX_IOV]` (common for both AM Short operations) for AM Short IOV? but need to set `sge_cnt`

if it is impossible:
what about defining `inl_am_iov_wr` + `inl_sge[UCT_IB_MAX_IOV]` (common for both AM Short operations) for AM Short IOV?